backend/events/apps.py

```diff
@@ -40,53 +40,82 @@ def get_event_loop():
         loop.close()
 
 
-def tuple_get_worker(tuple_type, _tuple):
-    if tuple_type == 'aggregatetuple':
-        return _tuple['worker']
-    return _tuple['dataset']['worker']
+def tuple_get_worker(event_type, _asset):
```

Comment:
minor: why adding a `_` prefix to asset (same everywhere)?

As far ass I know the `_` prefix on tuple is recommended because tuple is a python reserved keyword.

Reply:
Yes, only do a replace shortcut :)

Reply:
Done